client/static/js/game.js

```diff
@@ -0,0 +1,19 @@
+
+// TODO:  Generate a color map for N users, or create a static map
+var Game = {
```

Comment:
For `users`, it would probably be wise to have a  `player_id` on each `user` JSON object. That way we can identify each user without relying on their names